lib/LedgerSMB/Locale.pm

```diff
@@ -16,6 +16,12 @@ Locale support module for LedgerSMB.  Uses Locale::Maketext::Lexicon as a base.
 Returns a locale handle for accessing the other methods.  Inherited from
 Locale::Maketext.
 
+=item marktext
```

Comment:
All pre-existing functions document their parameters, meaning that this line should read:

`=item marktext ($string)`